commons/zenoh-config/src/lib.rs

```diff
@@ -292,6 +292,13 @@ validated_struct::validator! {
                 },
             },
         },
+        /// Configuration of the admin space.
+        pub adminspace: #[derive(Default)]
+        AdminSpaceConf {
+            /// Whether the admin space accepts config changes at runtime (false by default).
+            /// [unstable]: this configuration may change in the future.
+            changes: bool,
```

Comment:
`changes` is a bit of a weird name for this.

`writeable: bool` with the possibility to turn that into a structure that allows more precise filtering of the keys that are writeable sounds better to me

Reply:
Agreed for:
```json5
// [unstable]: this configuration may change in the future
adminspace: {
  permissions: {
    read: true,
    write: false,
  }
}
```

And for `zenohd` command line:
`--adminspace-permissions=[r|w|rw|none]` (default to `r`)